system/Database/Forge.php

```diff
@@ -531,19 +531,9 @@ public function createTable(string $table, bool $ifNotExists = false, array $att
     /**
      * @return bool|string
      */
-    protected function _createTable(string $table, bool $ifNotExists, array $attributes)
+    protected function _createTable(string $table, array $attributes)
```

Comment:
We could leave the parameter and mark it deprecated to prevent this from being a breaking change. Since this method is prefixed with an underscore I think we have slight plausible immunity. @kenjis What do you think?

Reply:
@MGatner Also its a protected function, shouldn't that have limit its use?

Reply:
@MGatner I added not to upgrade guide. I'm not sure exactly what you are looking for though.

Reply:
Don't change the method signature. Please add `@deprecated $ifNotExists is no longer used`.

Reply:
@kenjis I reverted the method signature as well as the protected properties.